cmd/root.go

```diff
@@ -3,13 +3,17 @@ package cmd
 import (
 	"fmt"
 
+	"github.com/gopinath-langote/1build/cmd/config"
 	parse "github.com/gopinath-langote/1build/cmd/config"
 	"github.com/gopinath-langote/1build/cmd/exec"
 	"github.com/gopinath-langote/1build/cmd/utils"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 )
 
+// FileFlag is a persistent (global) file flag for Root *cobra.Command
+var FileFlag string
```

Comment:
Why you need global flag?

Reply:
I was going off of an example from the `cobra` package. I'll remove this, as I'm using `StringP` method going forward.